src/unix/darwin.c

```diff
@@ -333,3 +333,29 @@ void uv_free_interface_addresses(uv_interface_address_t* addresses,
 
   uv__free(addresses);
 }
+
+int uv__tcp_enable_largesocket(uv_tcp_t *tcp) {
+  int rmem;
+  int wmem;
+  int r;
+
+  size_t size = sizeof(int);
+  if (sysctlbyname("net.inet.tcp.recvspace", &rmem, &size, NULL, 0) ||
+      rmem <= STDTCPWINDOW)
+    return UV_ENOSYS;
+  
+  if (sysctlbyname("net.inet.tcp.sendspace", &wmem, &size, NULL, 0) ||
```

Comment:
@gireeshpunathil yes I think that's possible. I think you should ask for access opening an issue in the https://github.com/nodejs/build repo.